indico/util/roles.py

```diff
@@ -0,0 +1,84 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+import csv
+
+from flask import flash
+
+from indico.core.errors import UserValueError
+from indico.modules.events.roles.forms import ImportMembersCSVForm
+from indico.modules.users import User
+from indico.util.i18n import _, ngettext
+from indico.util.string import to_unicode, validate_email
+from indico.web.flask.templating import get_template_module
+from indico.web.util import jsonify_data, jsonify_template
+
+
+class ImportRoleMembersMixin():
+
+    def import_members_from_csv(self, f):
+        """Import members from a CSV file into an event role."""
+        reader = csv.reader(f.read().splitlines())
+        emails = set()
+
+        for num_row, row in enumerate(reader, 1):
+            if len(row) != 1:
+                continue
+            try:
+                email = to_unicode(row[0]).strip().lower()
+            except ValueError:
+                raise UserValueError(_('Row {}: malformed CSV data').format(num_row))
+
+            if email and not validate_email(email):
+                raise UserValueError(_("Row {row}: invalid email address: {email}").format(row=num_row, email=email))
```

Comment:
replace `"` with `'`? Since two lines below you are using `'` (let's just be consistent)